configs/bundle_config.yaml

```diff
@@ -364,3 +370,29 @@ databundles:
     urls:
       protectedplanet: https://d1gam3xoknrgr2.cloudfront.net/current/WDPA_{month:s}{year:d}_Public_shp.zip
     output: [data/landcover/world_protected_areas/*]
+
+  # # tutorial bundle specific for Nigeria and Benin only, without gadm-like data
+  # bundle_tutorial_NGBJ:
```

Comment:
Probably, it would be handy to have a short explanation on the reason to comment-out this part with a link to https://github.com/pypsa-meets-earth/pypsa-earth/issues/1258

Reply:
Adding :)